src/util/CancellationHandle.h

```diff
@@ -148,9 +148,8 @@ class CancellationHandle {
                                          DESIRED_CANCELLATION_CHECK_INTERVAL}
                     << ", should be at most "
                     << ParseableDuration{DESIRED_CANCELLATION_CHECK_INTERVAL}
-                    << ". Stage: "
-                    << std::invoke(detailSupplier, AD_FWD(argTypes)...)
-                    << std::endl;
+                    << ". Checked at " << location.file_name() << ":"
+                    << location.line() << std::endl;
```

Comment:
This is where my comment about additional string infos and shortened filenames goes.